flink-libraries/flink-table/src/main/scala/org/apache/flink/table/catalog/InMemoryExternalCatalog.scala

```diff
@@ -98,6 +101,77 @@ class InMemoryExternalCatalog(name: String) extends CrudExternalCatalog {
     }
   }
 
+  @throws[ViewAlreadyExistException]
+  override def createView(
+    viewName: String,
+    view: String,
```

Comment:
I think it might be good to define a View class following the descriptor approach, though the only thing significant might be just a query as a string. I will update the doc accordingly.

Reply:
sounds good!

-